src/Microsoft.DotNet.Wpf/src/WpfGfx/codegen/mcg/Runtime/FileCodeSink.cs

```diff
@@ -155,12 +120,6 @@ protected virtual void Dispose(bool disposing)
                         _streamWriter = null;
                     }
 
-                    if (_fileCreated)
-                    {
-                        // If we created the file, we now need to Sd Add it
-                        TfFile(_filePath, "add");
-                    }
-
                     LogCreation();
```

Comment:
Probably, we can remove tf related code from LogCreation method as well.

Reply:
Yes, I'll remove the dead code.